### PR TITLE
Blog spec

### PR DESCRIPTION
**Pull request by mwafrika**
## Description

In this project, you will create all controllers and views for your Blog app. As you won't be processing any data yet, you will add simple placeholders for your views with plain HTML.

### General requirements

- [x] Followed the **Github flow**.
- [x] There are no linter errors.
- [x] The documentation is professional.

### JavaScript requirements

- [x] Followed list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### Project requirements

- [x] Add RSpec as a gem to your project.
- [x] Create a Request spec file for all your controllers.
- [x] Make sure that for each action you check:

> If response status was correct.
> If a correct template was rendered.
> If the response body includes correct placeholder text.

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,24 @@
-# Blog App
```
Comment (SajjadAhmad14) [inline README.md]: Please add a professional README to your project. A professional README contains project title, project description, and a set of instructions to run the app locally.

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
```
Comment (SajjadAhmad14) [inline spec/requests/posts_spec.rb:1]: All of the tests are failing. Kindly make sure that they pass all the test cases.
error


error1
error2
error3

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Mwafrika, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
PR has descriptive commit messages. :heavy_check_mark:
## To Improve 🙏
- [ ] Please add the `main/master` branch which is used for production releases in Git flow.
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (mwafrika) [reply README.md]: Alright, Thank you

**Comment by AshakaE** [comment]
Hi @mwafrika , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**